pkg/metrics/prometheus.go

```diff
@@ -180,6 +187,37 @@ func initStandardRegistry(config *types.Prometheus) Registry {
 		reg.entryPointOpenConnsGauge = entryPointOpenConns
 	}
 
+	if config.AddRouterLabels {
+		routerReqs := newCounterFrom(promState.collectors, stdprometheus.CounterOpts{
+			Name: routerReqsTotalName,
+			Help: "How many HTTP requests processed on an router, partitioned by status code, protocol, and method.",
```

Comment:
Could you update these descriptions to mention that it also get partitioned by `service`?

Reply:
Sure